lib/logstash/filters/fieldmap.rb

```diff
@@ -56,18 +60,61 @@ def filter(event)
       #  the key lenght, if not, explode
       split_src = event[@src_field].split(/#{@regex}/)
       @logger.debug? and @logger.debug("split_src is: ", :split_src => split_src)
+
+      if @text_qualifier
+        split_src_new_idx = split_src_orig_idx = 0
+        qualifier_start_idx = -1
+        split_src.map do |val|
+
+          val = val.strip   # this will cause problems if the text_qualifier is a space character
```

Comment:
please make a note in the comment above the text_qualifier parameter that it can't be a space.